bundles/org.openhab.binding.novafinedust/src/main/java/org/openhab/binding/novafinedust/internal/sds011protocol/SDS011Communicator.java

```diff
@@ -116,7 +131,12 @@ public boolean initialize(WorkMode mode, Duration interval)
             logger.debug("Will send command: {} ({})", HexUtils.bytesToHex(commandData), Arrays.toString(commandData));
         }
 
-        write(commandData);
+        try {
+            write(commandData);
+        } catch (IOException ioex) {
+            logger.debug("Got an exception while writing a command, will not try to fetch a reply for it.");
```

Comment:
You should include the exception message in the log so that it is easier to know what went wrong.